ktor-io/common/src/io/ktor/utils/io/ByteReadChannelOperations.kt

```diff
@@ -395,6 +396,32 @@ public suspend fun ByteReadChannel.discard(max: Long = Long.MAX_VALUE): Long {
 private const val CR: Byte = '\r'.code.toByte()
 private const val LF: Byte = '\n'.code.toByte()
 
+@JvmInline
+public value class LineEndingMode private constructor(private val mode: Int) {
```

Comment:
I could fix the problem only for chunked body, but it seems we should fix line endings in request/response parsing.